source/faq.txt

```diff
@@ -24,7 +24,7 @@ What Is the Difference Between "connectTimeoutMS", "socketTimeoutMS" and "maxTim
    * - connectTimeoutMS
      - | ``connectTimeoutMS`` is a :ref:`connection option <connection-options>` that sets the time in milliseconds for an individual connection from your connection pool to establish a TCP connection to the MongoDB server before timing out. Use the ``serverSelectionTimeoutMS`` option instead to fail when :node-api:`MongoClient.connect <MongoClient.html#.connect>` is unable to establish a connection with the MongoDB server.
```

Comment:
This could really stand a bit of a re-write. The first sentence goes on and on and should be broken up, IMO.

Reply:
I've rewritten it to clarify the main points and shorten it.